lib/platform/gitlab/index.ts

```diff
@@ -430,7 +451,18 @@ export async function createPr({
 export async function getPr(iid: number): Promise<Pr> {
   logger.debug(`getPr(${iid})`);
   const url = `projects/${config.repository}/merge_requests/${iid}?include_diverged_commits_count=1`;
-  const pr = (await api.get(url)).body;
+  const pr = (
+    await gitlabApi.getJson<
+      Pr & {
```

Comment:
We should have a helper function for mapping Pr to have proper types, see comments below